src/utils/strbuf.c

```diff
@@ -70,7 +70,7 @@ strbuf_release(strbuf_t *sb)
 {
 	if (sb->value != sb->buffer) {
 		/* Release the custom buffer. */
-		ASSERT(sb->bufsize > STRBUF_DEF_SIZE);
+		ASSERT(sb->bufsize >= STRBUF_DEF_SIZE);
```

Comment:
So I had this error:
```
./t_scoring
t_scoring: utils/strbuf.c:73: strbuf_release: Assertion `sb->bufsize > STRBUF_DEF_SIZE' failed.
Aborted
./t_strbuf
OK
./t_tokenize
t_tokenize: utils/strbuf.c:73: strbuf_release: Assertion `sb->bufsize > STRBUF_DEF_SIZE' failed.
Aborted
./t_utf8
OK
rm tests/helpers.o
```

Not entirely sure why needs to be strictly bigger, but I could see that sometimes it was the same size..